cli/src/command/account.rs

```diff
@@ -644,8 +664,24 @@ pub async fn node_info_command(account: &Account) -> Result<(), Error> {
 }
 
 /// `output` command
-pub async fn output_command(account: &Account, output_id: String) -> Result<(), Error> {
-    let output = account.get_output(&OutputId::from_str(&output_id)?).await;
+pub async fn output_command(account: &Account, selector: OutputSelector) -> Result<(), Error> {
+    let output = match selector {
+        OutputSelector::Id(id) => account.get_output(&id).await,
+        OutputSelector::Index(index) => {
+            let mut outputs = account.outputs(None).await?;
+            outputs.sort_by(|a, b| {
+                let cmp = b
+                    .metadata
+                    .milestone_index_booked()
+                    .cmp(&a.metadata.milestone_timestamp_booked());
+                match cmp {
+                    Ordering::Equal => a.output_id.cmp(&b.output_id),
+                    _ => cmp,
+                }
```

Comment:
Well maybe I start being annoying now but this is overkill, I'd just compare a tuple directly as suggested

Reply:
you're right ... fixed